### PR TITLE
clean up int_from_bytes

### PR DESCRIPTION
**Pull request by ofek**
7x speed-up and code is more readable

### CODE REVIEW

File: src/cryptography/utils.py [hunk 1]
```diff
@@ -48,18 +49,7 @@ def int_from_bytes(data, byteorder, signed=False):
...
-        result = 0
-
-        while len(data) > 0:
-            digit, = struct.unpack('>I', data[:4])
-            result = (result << 32) + digit
-            # TODO: this is quadratic in the length of data
-            data = data[4:]
-
-        return result
+        return int(codecs.encode(data, 'hex'), 16)
```
Comment (alex) [inline src/cryptography/utils.py]: Can this be written `data.encode("hex")`, instead of using the `codecs` module?

**Comment by alex** [comment]
DId this show up as a hotspot in a benchmark, or did you just happen across it?

**Comment by alex** [comment]
Jenkins, ok to test

File: src/cryptography/utils.py (hunk 1 above)
Comment (ofek) [inline src/cryptography/utils.py]: We can't because Python 3 is missing some built-in codecs [http://bugs.python.org/issue7475](url)

Comment (alex) [inline src/cryptography/utils.py]: This branch isn't used in python3 though, python3 always hits the `int.from_bytes` branch, right?

Comment (ofek) [inline src/cryptography/utils.py]: Yes, for 3.2+ (dunno about PyPy3). Would you like me to change it?

Comment (alex) [inline src/cryptography/utils.py]: Yeah, I think it's a bit cleaner.

**Pull request merged**